test/integration.test.js

```diff
@@ -19,7 +19,7 @@ describe('Integration testing generateFromFile() to make sure the result of the
     return path.resolve(mainTestResultPath, crypto.randomBytes(4).toString('hex'));
   };
 
-  jest.setTimeout(30000);
+  jest.setTimeout(60000);
```

Comment:
I had to increate timeout to 60s, because on windows integration tests always failed - first test create snapshot and the next one check that snapshot, so at the end all tests failed.